src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
Wow, I totally missed the fact that IndexRef type already holds information for the index order.  Nevermind!